README.md

```diff
@@ -98,6 +98,7 @@ Pass these options to the HoneycombWebSDK:
 | enabled | optional | boolean | `true` | Where or not to enable this auto instrumentation. |
 | lcp| optional| VitalOpts | `undefined` | Pass-through config options for web-vitals. See [ReportOpts](https://github.com/GoogleChrome/web-vitals?tab=readme-ov-file#reportopts).
 | lcp.applyCustomAttributes| optional| function | `undefined` | A function for adding custom attributes to core web vitals spans.
+| lcp.dataAttributes| optional| `string[]` | `undefined` | an array of data-* attribute names to filter. By default it will send all `data-*` attribute-value pairs with the key `lcp.element.data-someAttr`, `[]` will send none.
```

Comment:
I don't think this is how the values end up being added as attrs, according to the code each one is an attribute by itself with `.data.` prefix, can we change that here?

Reply:
💯 I wanted to make sure we were 👍 with [c3f6422](https://github.com/honeycombio/honeycomb-opentelemetry-web/pull/309/commits/c3f64222a27ce9bc4440f455a690695650371307) before updating all the docs. 😅